### ISSUE TITLE
bug with wiggle in info sections

### ISSUE THREAD

**Issue by sdake** [istio/istio.io#3900]
See:

https://deploy-preview-3891--preliminary-istio.netlify.com/docs/setup/kubernetes/prepare/platform-setup/minikube/

Mouse over
```
$  minikube config set vm-driver kvm2
```

cut and paste, print, etc also wiggle with the I.

**Comment by sdake** [comment]
PR where this was discovered: https://github.com/istio/istio.io/pull/3891

**Comment by geeknoid** [comment]
Yeah, I've known about this one for a while. I thought it was a cute buglet, so I didn't bother fixing it. I'll get there someday...
